apps/test/util/testUtils.js

```diff
@@ -291,3 +291,30 @@ export function throwOnConsoleWarnings() {
     firstError = null;
   });
 }
+
+/**
+ * Stub everything on window.Applab for every test in this describe block.
+ */
+export function stubWindowApplab() {
```

Comment:
Hmm, yeah - I was hoping the default 'stub everything if window.Applab exists' behavior would be flexible and we'd add to the default object substitute as needed, but maybe passing a default object in is better.  Or just replacing and restoring window.Applab entirely - I guess that's more unit-test friendly anyway.  Rethinking...